hi.js

```diff
@@ -0,0 +1,688 @@
+module.exports = `
+menu.btn.ranked रैंक का खेल
+menu.btn.shop दुकान
+menu.btn.social हब
+menu.btn.maps खेल
+menu.btn.mods मॉड
+menu.btn.settings सेटिंग
+menu.btn.customize रूप बदले
+menu.btn.invite आमंत्रण
+menu.btn.invite.clicked कॉपी
+menu.btn.join ज्वाइन
+menu.btn.host होस्ट खेल
+menu.btn.browser सर्वर
+generic.wait कृपया प्रतीक्षा करें...
+generic.loading लोड हो रहा है । ..
+generic.none कुछ भी नहीं
+generic.connecting कनेक्टिंग...
+generic.select चयन करें
+generic.delete हटाएं
+generic.save रूप बदले
+generic.load लोड
+generic.search खोजिए
+generic.leave छोड़ो
+generic.default डिफ़ॉल्ट
+generic.submit सबमिट करें
+generic.create बनाएँ
+generic.random रैंडम
+generic.failed असफल
+generic.success सफलता
+generic.name नाम
+generic.level स्तर
+generic.score स्कोर
+generic.off बंद
+generic.defeat हार
+generic.victory विजय
+generic.matchover मैच ओवर
+generic.matchaband मैच छोड़ दिया
+generic.follow फॉलो
+generic.unfollow अनफॉलो
+generic.followers अनुयायी
+generic.following निम्नलिखित
+generic.games खेल
+generic.wins जीत
+generic.lose घाटा
+generic.clan क्लैन
+generic.games.played खेले गये खेल
+generic.games.won खेल जीते
+generic.games.lost खेल हारे
+generic.timep कितना समय खेला
+generic.kills हत्या
+generic.deaths मौतें
+generic.kdr KDR
+generic.time समय
+generic.logout लॉगआउट
+generic.logout.sessions सभी सत्रों का लॉगआउट
+generic.username उपयोगकर्ता नाम
+generic.password पासवर्ड
+generic.register रजिस्टर
+generic.login लॉगिन
+generic.accuracy सटीकता
+player.waiting खिलाड़ियों का इंतजार
+player.died तुम मर गए
+player.killed द्वारा मारे गए
+player.killstreak {0} एक {1} मार लकीर पर
```

Comment:
Let's revert this one back to English.

The grammar will appear as:

`You're {1} मार लकीर पर`

or

`Player is एक {1} मार लकीर पर`

Reply:
https://github.com/mty22/krunker.io-translations/blob/main/en.js#L64

Reply:
The actual meaning of the word "लकीर" is "a straight drawn line". Another Google Translation Error. 

Better to keep it as kill streak only

Reply:
So we keep it as killstreak only? Or should I change it to what Matt [ `You're {1} मार लकीर पर` ] has asked to revert to?

Reply:
Just revert it to English as it needs to be accurate.

Google Translate, whilst being a fantastic tool, really should not be used in projects like this. It does not have true understanding of each word as they're not full sentences, thus no context to work from without having knowledge of the game.